flatten-docs.py

```
@@ -0,0 +1,287 @@
+        total_changes = rewrite_html_files_parallel(html_files, version_dir, module_dir, flattened_relative)
+        print(f"  Updated {total_changes} links in {len(html_files)} files")
+
+        # Move module directory to flattened location
+        target_dir = version_dir / flattened_relative
+        if target_dir.exists():
+            print(f"  Warning: {target_dir} already exists, skipping move")
+        else:
+            try:
+                shutil.move(str(module_dir), str(target_dir))
+                print(f"  Moved to {flattened_relative}")
+            except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)